<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrashReportWidget</class>
 <widget class="QWidget" name="CrashReportWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeL report</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="descrLabel">
     <property name="text">
      <string>What were you doing when the crash occurred?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="reportLabel">
     <property name="text">
      <string>Contents of the report ( automatically generated )</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="descriptionEdit"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="emailCB">
     <property name="text">
      <string>Email me if you have further questions, or updates on this issue</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLineEdit" name="emailEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enter your email address here</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="reportEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
